public/main.ts

```diff
@@ -93,8 +107,8 @@ class Kingdom {
     army = new Army();
     farms: Map<Crop, Farm> = new Map();
 
-    constructor(name: string, initialPopulation: number){
-        this.idlePopulation = initialPopulation;
+    constructor(name: string, idlePopulation: number){
+        this.idlePopulation = idlePopulation;
         this.name = name;
     }
```

Comment:
```suggestion
    constructor(public name: string, public idlePopulation: number){}
```

Typescript allows this. [Source](https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties)